### PR TITLE
Add proxy 2.6 jobs

### PR DESCRIPTION
**Pull request by jwendell**

### CODE REVIEW

File: ci-operator/config/maistra/proxy/maistra-proxy-maistra-2.6.yaml [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+      LOCAL_JOBS: "30"
+      LOCAL_RAM_RESOURCES: "61440"
+      MAISTRA_BUILDER_IMAGE: registry.ci.openshift.org/ci/maistra-builder:2.6
+      MAISTRA_NAMESPACE: maistra-e2e-test
+      MAISTRA_SC_POD: maistra-proxy-test-pod
+    test:
+    - as: copy-src
+      cli: latest
+      commands: |
+        tar cf - . | oc exec -i -n "${MAISTRA_NAMESPACE}" "${MAISTRA_SC_POD}" -- tar xf - -C /work/
```
Comment (yxun) [inline ci-operator/config/maistra/proxy/maistra-proxy-maistra-2.6.yaml:40]: do you need to mention the container name in the MAISTRA_SC_POD ?

Comment (jwendell) [reply ci-operator/config/maistra/proxy/maistra-proxy-maistra-2.6.yaml:40]: I have no idea. I just copied 2.5 jobs :/

File: ci-operator/config/maistra/proxy/maistra-proxy-maistra-2.6.yaml [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+      - name: CI
+      - name: LOCAL_CPU_RESOURCES
+      - name: LOCAL_RAM_RESOURCES
+      - name: LOCAL_JOBS
+      from: src
+      resources:
+        requests:
+          cpu: 100m
+          memory: 100Mi
+      timeout: 8h0m0s
```
Comment (yxun) [inline ci-operator/config/maistra/proxy/maistra-proxy-maistra-2.6.yaml:73]: In the past, I remembered the longest timeout value was 4h. We can try 8h here and see if it works.

File: ci-operator/config/maistra/proxy/maistra-proxy-maistra-2.6.yaml (hunk 1 above)
Comment (yxun) [reply ci-operator/config/maistra/proxy/maistra-proxy-maistra-2.6.yaml:40]: ok, I see. I forgot how I wrote those before

**Review by yxun — APPROVED** [review]

**Pull request merged**